### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by pgrimaud**
Hello,

This PR will fix : 

- 2 typos in `README.md`
- 1 typo in `test/integration-serial/serial_task_run_test.rb`

Cheers! :robot:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ official compatibility chart below.
 
 *Environment variables:*
 
-- `$KUBECONFIG`: points to one or multiple valid kubeconfig files that include the context you want to deploy to. File names are separated by colon for Linux and Mac, and semi-colon for Windows. If ommitted, will use the Kubernetes default of `~/.kube/config`.
+- `$KUBECONFIG`: points to one or multiple valid kubeconfig files that include the context you want to deploy to. File names are separated by colon for Linux and Mac, and semi-colon for Windows. If omitted, will use the Kubernetes default of `~/.kube/config`.
```
Comment (timothysmith0609) [inline README.md:106]: While we're here...

>If omitted, will use the Kubernetes default....

Change to:
_If omitted, Krane will use the Kubernetes default..._

File: README.md [hunk 2]
```diff
@@ -544,7 +544,7 @@ krane render -f ./path/to/template/dir/template.yaml.erb > template.yaml
 - `--bindings=BINDINGS`: Makes additional variables available to your ERB templates. For example, `krane render --bindings=color=blue size=large -f some-template.yaml.erb` will expose `color` and `size` to `some-template.yaml.erb`.
 - `--current-sha`: Expose SHA `current_sha` in ERB bindings
 
-You can add additional variables using the `--bindings=BINDINGS` option which can be formated as a string, JSON string or path to a JSON or YAML file. Complex JSON or YAML data will be converted to a Hash for use in templates. To load a file the argument should include the relative
+You can add additional variables using the `--bindings=BINDINGS` option which can be formatted as a string, JSON string or path to a JSON or YAML file. Complex JSON or YAML data will be converted to a Hash for use in templates. To load a file the argument should include the relative
 file path prefixed with an `@` sign. An argument error will be raised if the string argument cannot be parsed, the referenced file does not include a
  valid extension (`.json`, `.yaml` or `.yml`) or the referenced file does not exist.
```
Comment (timothysmith0609) [inline README.md:549]: - >To load a file the argument should include... -> To load a file, the argument... (missing comma)

- The line breaks here are quite strange, can we clear those up while we're here, too?

Comment (pgrimaud) [reply README.md]: I added the missing coma and fixed the line breaks.

File: README.md (hunk 1 above)
Comment (pgrimaud) [reply README.md]: Changed as requested 😃

**Pull request merged**